Samples/SimpleIslandApp/SimpleIslandApp.cpp

```diff
@@ -0,0 +1,224 @@
+﻿// SimpleIslandApp.cpp : Defines the entry point for the application.
+//
+
+#include "pch.h"
+#include "SimpleIslandApp.h"
+
+#include <App.xaml.h>
+#include <MainPage.h>
+
+namespace winrt
+{
+    using namespace winrt::Microsoft::UI;
+    using namespace winrt::Microsoft::UI::Dispatching;
+    using namespace winrt::Microsoft::UI::Xaml;
+    using namespace winrt::Microsoft::UI::Xaml::Hosting;
+    using namespace winrt::Microsoft::UI::Xaml::Markup;
+}
+
+#define MAX_LOADSTRING 100
+
+// Global Variables:
```

Comment:
are these globals all necessary?
HINSTANCE is equivalent to GetModuleHandle(NULL) 
szTitle & szWindowClass could be made InitInstance statics (yes still global, but a bit tidier)
what about an abstraction for the window to allow m_button1, m_button2?

I know we want a bare bones sample with few dependencies - just wonder whether we can do better than Petzold style